index.html

```diff
@@ -3986,42 +3966,35 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
           </aside>
 
           <aside class="example">
-            Example <span class="chunk">mDCv</span> chunk mastering display maximum 
-          luminance using traditional <a>SDR</a> (for example [[ITU-R-BT.709]]) shading techniques at 100 cd/m<sup>2</sup> 
-
-            (described in [[ITU-R-BT.2035]]):
-            <table id="mDCv-chunk-max-luminance-example3" class="numbered simple">
+            Example <span class="chunk">mDCv</span> chunk mastering display maximum
+          luminance for [[SRGB]] <a>SDR</a>:
+            <table id="mDCv-chunk-sdr-max-luminance-example" class="numbered simple">
               <tr>
                 <th>Actual value</th>
                 <th>Stored value</th>
               </tr>
 
               <tr>
-                <td>100 cd/m<sup>2</sup></td>
-                <td>1000000</td>
+                <td>80 cd/m<sup>2</sup></td>
+                <td>800000</td>
               </tr>
             </table>
           </aside>
 
           <aside class="example">
-
-            Example <span class="chunk">mDCv</span> chunk mastering display maximum 
-          luminance using "single-master" <a>SDR</a> (for example [[ITU-R-BT.709]]) shading techniques at 203 cd/m<sup>2</sup> 
-
-            (described in ITU-R BT.2408 Annex Y):
-            <table id="mDCv-chunk-max-luminance-example4" class="numbered simple">
+            Example mDCv chunk mastering display minimum luminance for [[SRGB]] <a>SDR</a>:
+            <table id="mDCv-chunk-sdr-min-luminance-example" class="numbered simple">
               <tr>
                 <th>Actual value</th>
                 <th>Stored value</th>
               </tr>
 
               <tr>
-                <td>203 cd/m<sup>2</sup></td>
-                <td>2030000</td>
+                <td>0.05 cd/m<sup>2</sup></td>
```

Comment:
Should we use  the [VESA Certified DisplayHDR Performance Levels](https://displayhdr.org/)? If so either 0.1 / 500 or 0.05 / 1000

Reply:
Agreed - makes sense to use those values instead.

Reply:
These examples are for SDR, and the value is already 0.05 so I'm unsure what needs changing.